src/util/FitsComparatorCMD.cpp

```diff
@@ -0,0 +1,86 @@
+//
+// Created by Samuel Cappon on 3/9/20.
+//
+
+#include <util/FitsComparator.h>
+#include <core/Exceptions.h>
+#include <iostream>
+#include <boost/program_options.hpp>
+#include <boost/program_options/errors.hpp>
+
+namespace po = boost::program_options;
+
+main(int ac, char *av[]) {
```

Comment:
it is fine to use main like this but normally preferred way is `int main`. Then it follows the regular function prototype
`return_type function_name()`